src/DIRAC/ConfigurationSystem/Client/SyncPlugins/CERNLDAPSyncPlugin.py

```diff
@@ -76,6 +76,7 @@ def _getUserInfo(self, commonName):
     )
     if not status:
       raise ValueError("Bad status from LDAP search: %s" % result)
+    response = list(response)
```

Comment:
why do we need the cast here ?

Reply:
Pylint thought `response` was a generator and can't be indexed a few lines lower

Reply:
so why not just disabling the rule of the offending line ? In any case, a comment would surely help, otherwise it looks like it's part of t he logic

Reply:
Somehow I persuaded myself `pylint` was correct but it wasn't. Fixed.